src/main/steps/terms-and-conditions/content.test.ts

```diff
@@ -7,163 +7,110 @@ jest.mock('../../app/form/validation');
 
 const en = {
   title: 'Terms and conditions',
-  thisPage:
-    'By using this service you’re agreeing to these terms of use. This includes the <a class="govuk-link" href="/privacy-policy">privacy policy</a>.',
-  termsAndConditionsUsers: 'Terms and conditions for professional users',
-  termsAndConditionsProfessional:
-    "If you are a professional user, you'll be asked to agree to the additional terms and conditions of the service after you create a MyHMCTS account.",
+  byAccessingThisService:
+    'By accessing this service (‘the Service’), you’re agreeing to the following terms of use. This includes the <a class="govuk-link" href="/privacy-policy">privacy policy</a>.',
```

Comment:
‘the Service’ - it should be service name, i.e. Private law. You can refer to cookie banner page to know about how we can put service name.

Reply:
changed to private law service